docs/source/examples/checks/distribution/trust_score_comparison.ipynb

```diff
@@ -2,106 +2,136 @@
  "cells": [
```

Comment:
<p>alternative phrasing (fixing some typos etc.) for last paragraph:</p><p>Furthermore, we may notice some other common characteristics, such as the fact that <code>worst texture</code> and <code>mean texture</code> both seem to be lower in the top scoring samples, while the worst scoring samples have <code>worst texture</code> and <code>mean texture</code>  values, both features with high feature importance for the AdaBoost model. Might it be that high texture samples are getting worse predictions by the model? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>